scripts/build_nrf_unix.sh

```diff
@@ -7,55 +7,16 @@ PROJECT_ROOT=$LF_BIN_DIRECTORY/..
 echo "starting NRF generation script into $LF_SOURCE_GEN_DIRECTORY"
 echo "pwd is $(pwd)"
 
-# Copy platform into /core
-cp $PROJECT_ROOT/platform/lf_nrf52_support.c $LF_SOURCE_GEN_DIRECTORY/core/platform/
-cp $PROJECT_ROOT/platform/lf_nrf52_support.h $LF_SOURCE_GEN_DIRECTORY/core/platform/
-cp $PROJECT_ROOT/platform/platform.h $LF_SOURCE_GEN_DIRECTORY/core/
-cp $PROJECT_ROOT/platform/reactor.c $LF_SOURCE_GEN_DIRECTORY/core/
-cp $PROJECT_ROOT/platform/reactor_common.c $LF_SOURCE_GEN_DIRECTORY/core/
-
-# Copy platform into /include/core
-# TODO: Why are there two generated core dirs
-cp $PROJECT_ROOT/platform/lf_nrf52_support.c $LF_SOURCE_GEN_DIRECTORY/include/core/platform/
-cp $PROJECT_ROOT/platform/lf_nrf52_support.h $LF_SOURCE_GEN_DIRECTORY/include/core/platform/
-cp $PROJECT_ROOT/platform/platform.h $LF_SOURCE_GEN_DIRECTORY/include/core/
-cp $PROJECT_ROOT/platform/reactor.c $LF_SOURCE_GEN_DIRECTORY/include/core/
-cp $PROJECT_ROOT/platform/reactor_common.c $LF_SOURCE_GEN_DIRECTORY/include/core/
-
 # Copy the lib directory.
 cp -r $PROJECT_ROOT/lib/* $LF_SOURCE_GEN_DIRECTORY/lib
+# Copy makefile
+cp $PROJECT_ROOT/platform/Makefile $LF_SOURCE_GEN_DIRECTORY/
```

Comment:
The convenient thing with having a script that generated the makefile is that we could easily find the root of the project and thus get a correct path to the NRF SDKs. An alternative is to pass the project root as an argument to make.  A better alternative might be for this script to open Makefil and add add a line to the top e.g.
```
PROJECT_ROOT=$LF_BIN_DIRECTORY/..
```

Then save it into the new location.